src/assets/wise5/authoringTool/edit-branch/edit-branch.component.ts

```diff
@@ -215,11 +217,20 @@ export class EditBranchComponent extends AbstractBranchAuthoringComponent {
   protected async submit(): Promise<void> {
     this.submitting = true;
     const params = this.getBranchParams();
+    this.createNewPaths(this.branchPaths, params);
```

Comment:
We can get ```this.branchPaths``` in the function?
```suggestion
    this.createNewPaths(params);
```